### ISSUE TITLE
Promote AdoptOpenJDK Version jdk-11.0.9+11

### ISSUE THREAD

**Issue by andrew-m-leonard** [AdoptOpenJDK/TSC#182]
Java Version:

JVM:
- [x] OpenJ9
- [x] Hotspot

@AdoptOpenJDK/tsc please can somebody +1 this request?

**Comment by andrew-m-leonard** [comment]
**openj9 triage:**
aarch64 sanity.functional:
  - Existing issue: https://github.com/eclipse/openj9/issues/10843
  - Existing issue: https://github.com/eclipse/openj9/issues/10741

aarch64 sanity.openjdk:
  - Existing issue: https://github.com/eclipse/openj9/issues/10597

**Comment by karianna** [comment]
TRSS [link](https://trss.adoptopenjdk.net/buildDetail?parentId=5f91d41111d26031c124fbfc&testSummaryResult=failed&buildNameRegex=%5ETest) 
Build URL https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1225/ 
Started by user George Adams 
### ⚠️  [jdk11u-aix-ppc64-hotspot](https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-aix-ppc64-hotspot/735/) has a build result of FAILURE ⚠️
### ⚠️  [jdk11u-windows-x86-32-hotspot](https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-windows-x86-32-hotspot/718/) has a build result of FAILURE ⚠️

[**Test_openjdk11_hs_extended.system_x86-64_mac**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_extended.system_x86-64_mac/380/) 
HCRLateAttachWorkload_previewEnabled_0 => [deep history 2/13 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f92520e11d26031c12cff90) - known infra issue https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1274 (test passes on macos1010, fails on later versions)

[**Test_openjdk11_hs_sanity.external_s390x_linux**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.external_s390x_linux/27/) 

[**Test_openjdk11_hs_sanity.external_s390x_linux_openliberty-mp-tck**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.external_s390x_linux_openliberty-mp-tck/20/) 
openliberty_microprofile_tck_0 => [deep history 0/9 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f92a24011d26031c1302432)

**Comment by gdams** [comment]
JDk11 re-run on failing platforms + linux (x64/aarch64) https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1229/

**Comment by adamfarley** [comment]
OpenJ9 Release Pipelines:
Linux 64bit + XL: https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1227/
Aarch64 + Aarch64XL: https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1228/
Aarch64 + Aarch64XL (testless original run): https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1222/
Windows 64bit: https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1224/
Mac + XL, Windows (XL only),  AIX ppc64, ppc64le + XL, s390xLinux + XL (plus testless linux builds and a broken windows build): https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1221/

**Comment by andrew-m-leonard** [comment]
OpenJ9 Test results: 
- Aarch64/XL: https://trss.adoptopenjdk.net/resultSummary?parentId=5f92aeb011d26031c1303d1d
  - All existing issues
- xLinux: https://trss.adoptopenjdk.net/resultSummary?parentId=5f92aeb011d26031c1303d1f
   - All green
- Win64: https://trss.adoptopenjdk.net/resultSummary?parentId=5f915a5411d26031c11fcc83
   - All green
- others: https://trss.adoptopenjdk.net/resultSummary?parentId=5f913a1211d26031c11e8f54
   - All green

**Comment by andrew-m-leonard** [comment]
OpenJ9 pipelines to Publish:
- https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1221/
- https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1222/
- https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1224/

**Comment by gdams** [comment]
JDK 11 re-run

TRSS [link](https://trss.adoptopenjdk.net/buildDetail?parentId=5f92c0a011d26031c1306eb6&testSummaryResult=failed&buildNameRegex=%5ETest) 
Build URL https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1229/ 
Started by user George Adams 
### ⚠️  [jdk11u-aix-ppc64-hotspot](https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-aix-ppc64-hotspot/737/) has a build result of FAILURE ⚠️

[**Test_openjdk11_hs_sanity.system_x86-64_linux**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.system_x86-64_linux/397/) 
TestJlmRemoteClassAuth_0 => [deep history 3/4 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f9300ca11d26031c130f63f) 
TestJlmRemoteClassNoAuth_0 => [deep history 3/4 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f9300ca11d26031c130f64c) 
TestJlmRemoteMemoryAuth_0 => [deep history 3/4 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f9300ca11d26031c130f658) 
TestJlmRemoteMemoryNoAuth_0 => [deep history 3/4 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f9300ca11d26031c130f664) 
TestJlmRemoteNotifierProxyAuth_0 => [deep history 3/4 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f9300ca11d26031c130f671) 
TestJlmRemoteThreadAuth_0 => [deep history 0/1 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f9300ca11d26031c130f640) 
TestJlmRemoteThreadNoAuth_0 => [deep history 0/1 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f9300ca11d26031c130f64d) 
- TestJlm* are recently re-enabled tests (not run for many past releases), non-blocking

[**Test_openjdk11_hs_sanity.external_x86-64_linux**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.external_x86-64_linux/496/) - non-blocking (parent for quarkus_quickstarts/openliberty-mp-tck below)

[**Test_openjdk11_hs_sanity.openjdk_x86-32_windows**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.openjdk_x86-32_windows/363/) 
jdk_lang_native_win_0 => [deep history 0/14 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f92d7e511d26031c130be5a) - created issue https://github.com/AdoptOpenJDK/openjdk-tests/issues/2029

[**Test_openjdk11_hs_special.openjdk_aarch64_linux**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_special.openjdk_aarch64_linux/1/) - new/non-blocking

[**Test_openjdk11_hs_extended.perf_aarch64_linux**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_extended.perf_aarch64_linux/1/) - new/non-blocking

[**Test_openjdk11_hs_special.functional_aarch64_linux**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_special.functional_aarch64_linux/1/) - new/non-blocking

[**Test_openjdk11_hs_extended.system_x86-32_windows**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_extended.system_x86-32_windows/26/) 
MathLoadTest_all_0 => [deep history 0/13 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f92d7e511d26031c130be2d) 
LockingLoadTest_0 => [deep history 0/13 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f92d7e511d26031c130be32) - longstanding/non-blocking

[**Test_openjdk11_hs_sanity.external_x86-64_linux_openliberty-mp-tck**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.external_x86-64_linux_openliberty-mp-tck/480/) 
openliberty_microprofile_tck_0 => [deep history 0/4 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f93015111d26031c130f84f) - non-blocking

[**Test_openjdk11_hs_sanity.external_x86-64_linux_quarkus_quickstarts**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.external_x86-64_linux_quarkus_quickstarts/100/) 
quarkus_quickstarts_test_0 => [deep history 3/4 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f93015211d26031c130f858) - non-blocking

[**Test_openjdk11_hs_extended.openjdk_aarch64_linux**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_extended.openjdk_aarch64_linux/2/)  - new/non-blocking
jvm_compiler_0 => [deep history 0/1 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f93014c11d26031c130f7de) 
jdk_awt_0 => [deep history 0/1 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f93014c11d26031c130f7df) 
jdk_beans_0 => [deep history 0/1 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f93014c11d26031c130f7db) 


[**Test_openjdk11_hs_special.system_aarch64_linux**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_special.system_aarch64_linux/1/) - new/non-blocking

[**Test_openjdk11_hs_special.perf_aarch64_linux**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_special.perf_aarch64_linux/1/) - new/non-blocking

**Comment by andrew-m-leonard** [comment]
OpenJ9 shipped.

**Comment by gdams** [comment]
AIX rebuild:

TRSS [link](https://trss.adoptopenjdk.net/buildDetail?parentId=5f92fd8e11d26031c130f2d9&testSummaryResult=failed&buildNameRegex=%5ETest) 
Build URL https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1230/ 
Started by user George Adams 

[**Test_openjdk11_hs_extended.system_ppc64_aix**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_extended.system_ppc64_aix/34/) 
MathLoadTest_all_0 => [deep history 0/13 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f931dde11d26031c1310097) 
LockingLoadTest_0 => [deep history 0/13 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f931dde11d26031c131009c)

**Comment by smlambert** [comment]
Hotspot looks fine, all failures except one are known non-blockers.

The only unknown/untriaged failure of the hotspot set is: 

[**Test_openjdk11_hs_sanity.openjdk_x86-32_windows**](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.openjdk_x86-32_windows/363/) 
jdk_lang_native_win_0 => [deep history 0/14 passed](https://trss.adoptopenjdk.net/deepHistory?testId=5f92d7e511d26031c130be5a) - created issue https://github.com/AdoptOpenJDK/openjdk-tests/issues/2029

Appears to be a relatively recent test case addition into the openjdk group, see notes in https://github.com/AdoptOpenJDK/openjdk-tests/issues/2029#issuecomment-715525897 failing for at least 75 days with:

java.lang.UnsatisfiedLinkError: 'long CheckHandles.getProcessHandleCount()'
at CheckHandles.getProcessHandleCount(Native Method)
at CheckHandles.main(CheckHandles.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:298)
at java.base/java.lang.Thread.run(Thread.java:834)

Do not know if it has ever passed on win32 platform.  Grinder: https://ci.adoptopenjdk.net/job/Grinder/4305/

**Comment by gdams** [comment]
@smlambert thanks for the update, I'll proceed with shipping 11

**Comment by gdams** [comment]
JDK11 hotspot shipped

**Comment by gdams** [comment]
OpeJ9 macOS re-run:

TRSS [link](https://trss.adoptopenjdk.net/buildDetail?parentId=5f93454011d26031c1312bec&testSummaryResult=failed&buildNameRegex=%5ETest) 
Build URL https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1231/ 
Started by user George Adams

NO FAILURES

**Comment by sxa** [comment]
Shenandoah missing from HotSpot JDK11, Fixed in https://github.com/AdoptOpenJDK/openjdk-build/pull/2177.
Re-running at https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/1233/console with `adoptBuildNumber=1`

Checked WIn/x32 and it accepts the option:
```
C:\Users\sxa\jdk-11.0.9+11-jre\bin>java -XX:+UseShenandoahGC -Xlog:gc -version
[0.036s][info][gc] Heuristics ergonomically sets -XX:+ExplicitGCInvokesConcurrent
[0.038s][info][gc] Heuristics ergonomically sets -XX:+ShenandoahImplicitGCInvokesConcurrent
[0.042s][info][gc] Using Shenandoah
openjdk version "11.0.9" 2020-10-20
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.9+11)
OpenJDK Client VM AdoptOpenJDK (build 11.0.9+11, mixed mode)
[0.294s][info][gc] Cancelling GC: Stopping VM
```
*Summary:*

- Known issues on Windows/x32: PathLoadTest_all, LockingLoadTest, jdk_lang_native_win
- macos: core file detected during a previous run of [sanity.system](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.system_x86-64_mac/389/console) (Not created during current run). Have marked affected machine offline and rerunning at https://ci.adoptopenjdk.net/job/Grinder/4311/
- Windows/64 - [sanity.perf job](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_sanity.perf_x86-64_windows/321/) seems to have got stuck, re-running in [Grinder](https://ci.adoptopenjdk.net/job/Grinder/4310 and it passed ok
- Linux/x64: Usual failure on openliberty TCK
- LInux/aarch64: All required jobs complete but it's still running additional tests which can be canceled [last one](https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_extended.openjdk_aarch64_linux/3/).

**Comment by sxa** [comment]
@AdoptOpenJDK/tsc Request to ship the above Shenandoah rebuild  of JDK11/HotSpot please? Summary of test issues in previous comment

TL;DR: WIndows perf hang, rerun ok. macos rogue core file on machine, re-run elsewhere still in progress. ~Approvals will be conditional on that and the subsequent installer jobs completing of course.~ (All complete)

**Comment by sxa** [comment]
Versions with Shenandoah now available as [11.0.9+11.1](https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/tag/jdk-11.0.9%2B11.1). Everything now complete with this release, so closing. Retrospective for the releases are in #181
